### ISSUE TITLE
Enhancement request: provide keyboard navigation to new block embed menu

### ISSUE THREAD

**Issue by cobblepot1** [RoamJS/workbench#336]
Clicking on a rendered embedded block (i.e., not the block reference, but the block content associated with that reference) now pops up a menu with several important functions, starting with "Jump to block". The keyboard-centric user cannot access that menu without a mouse. Please add a way to access all of those functions, including the functions on the "Replace with" submenu, using keyboard only. Many thanks!

**Comment by dvargasfuertes** [comment]
Roam42 Issue Log Audit - Could add to mouseless if user interest resurfaces

**Comment by cobblepot1** [comment]
Still interested!

**Comment by dvargasfuertes** [comment]
should now be supported by roam natively
